lxd/network/openvswitch/ovn.go

```diff
@@ -116,7 +116,7 @@ type OVNDHCPv6Opts struct {
 // OVNSwitchPortOpts options that can be applied to a swich port.
 type OVNSwitchPortOpts struct {
 	MAC          net.HardwareAddr   // Optional, if nil will be set to dynamic.
-	IPs          []net.IP           // Optional, if empty IPs will be set to dynamic.
```

Comment:
I wonder if it would be clearer to just have a boolean for dynamic IPs and then error if `(o.DynamicIPs && len(o.IPs) > 0) || (!o.DynamicIPs && len(o.IPs) == 0)`

Reply:
Hmm looking further in the PR this might actually complicate logic elsewhere :thinking:

Reply:
yeah maybe, ill merge this to see if the lxd-ci tests are fixed, and revisit. thanks